package.json

```diff
@@ -58,12 +58,14 @@
   "devDependencies": {
     "async": "^2.0.0",
     "colors": "^1.1.2",
+    "combined-stream": "^1.0.5",
```

Comment:
I think `combined-stream` should be under regular `dependencies`, not `devDependencies`. `devDependencies` don't get installed in a production environment, so this could lead to big trouble.